app/controllers/application_controller.rb

```diff
@@ -10,4 +10,17 @@ def set_locale
   def default_url_options
     {locale: I18n.locale}
   end
+
+  def break_out
+    redirect_to root_path
+    flash.alert = t "warning"
+  end
+
+  def verify_admin
+    if user_signed_in?
```

Comment:
```suggestion
    return if user_signed_in? && current_user.admin?
    break_out
```
done :v 4 dòng dưới có thể xóa =))

Reply:
Vâng ạ :)